DESCRIPTION

```diff
@@ -49,4 +49,4 @@ Imports:
 VignetteBuilder: knitr
 URL: https://github.com/sailuh/kaiaulu
 BugReports: https://github.com/sailuh/kaiaulu/issues
-RoxygenNote: 7.2.3
+RoxygenNote: 7.3.2
```

Comment:
Do not change this

Reply:
Should I revert `RoxygenNote` back to `7.2.3` in my next commit?

Reply:
It should keep the original, yes. Each of you in the group is changing it.